release/api/v1alpha1/eksarelease.go

```diff
@@ -10,5 +10,6 @@ const EKSAReleaseKind = "EKSARelease"
 
 // Generates the naming convention of EKSARelease from a version.
 func GenerateEKSAReleaseName(version string) string {
+	version = strings.ReplaceAll(version, "+", "-")
 	return fmt.Sprintf("eksa-%s", strings.ReplaceAll(version, ".", "-"))
```

Comment:
is there a way to reverse this transformation?
i wonder if we should use a diff algorithm so we can go back and forth
let me know if you don't think this is doable

Reply:
There should only ever be a "+" in front of build so to reverse this transformation, we could just turn the last "." into a "+".

Reply:
Semver can also have a "-" after the three numbers
What do you think about replacing "+" with "-plus-"?